addons/mail/static/src/discuss/core/common/discuss_notification_settings_client_action.js

```diff
@@ -1,12 +1,18 @@
-import { Component } from "@odoo/owl";
+import { Component, xml } from "@odoo/owl";
 import { registry } from "@web/core/registry";
 
 import { DiscussNotificationSettings } from "@mail/discuss/core/common/discuss_notification_settings";
 
 export class DiscussNotificationSettingsClientAction extends Component {
     static components = { DiscussNotificationSettings };
     static props = ["*"];
-    static template = "mail.DiscussNotificationSettingsClientAction";
+    static template = xml`
+        <div class="o-mail-DiscussNotificationSettingsClientAction mx-3 my-2">
+            <DiscussNotificationSettings/>
+        </div>
+    `;
+
+    "mail.DiscussNotificationSettingsClientAction";
```

Comment:
`"mail.DiscussNotificationSettingsClientAction"` Can be removed, no need named template when using inline template definition.